Perpetuum.DataDumper/Views/RobotDataView.cs

```diff
@@ -0,0 +1,142 @@
+
+            var allSlotFlags = head.ED.Options.SlotFlags
+                                    .Concat(chassis.ED.Options.SlotFlags)
+                                    .Concat(legs.ED.Options.SlotFlags).ToList();
+
+            var typeFlags = new List<SlotFlags>();
+
+            foreach (var slotFlags in allSlotFlags) {
+                var currentSlotTypes = EnumHelper<SlotFlags>.MaskToList((SlotFlags)Convert.ToInt64(slotFlags)).ToList()
+                                    .Intersect(SLOT_TYPE_FLAGS).ToList();
+
+                typeFlags.AddRange(currentSlotTypes);
+            }
+            
+
+            // main Stats
+            Speed = (double)input.Properties.SingleOrDefault(x=> x.Field == AggregateField.speed_max).Value * 36; // needs to be multiplied by 36
+            Armor = input.ArmorMax;
+            Accumulator = input.CoreMax;
+            Cargo = (double) inventory.GetCapacityInfo()["capacity"];
+            Cpu = input.Cpu;
+            // The Max property is what we want, but it's not public and since this is a fake robot it will equal max
+            Reactor = input.PowerGrid;
+            
+            if (input.Slope == 4) {
+                SlopeCapacity = 45;
+            } else if (input.Slope == 5) {
+                SlopeCapacity= 51;
+            } else if (input.Slope == 6) {
+                SlopeCapacity= 56;
+            } else {
+                SlopeCapacity = input.Slope;
+            }
+
+            // Slots
+            SlotsHead = head.MaxSlots;
+            SlotsChassis = chassis.MaxSlots;
+            SlotsLeg = legs.MaxSlots;
+            SlotsGuns = typeFlags.Where(x => x == SlotFlags.turret).Count();
+            SlotsMissiles = typeFlags.Where(x => x == SlotFlags.missile).Count();
+            SlotsIndustrial = typeFlags.Where(x => x == SlotFlags.industrial).Count();
+            SlotsMisc = typeFlags.Where(x => x == SlotFlags.ew_and_engineering).Count();
+
+            // Defense
+            SurfaceSize = input.SignatureRadius;
+            ResistanceChem = input.GetPropertyModifier(AggregateField.resist_chemical).Value;
+            ResistanceKinetic = input.GetPropertyModifier(AggregateField.resist_kinetic).Value;
+            ResistanceSeismic = input.GetPropertyModifier(AggregateField.resist_explosive).Value;
+            ResistanceThermal = input.GetPropertyModifier(AggregateField.resist_thermal).Value;
+            ResistanceDemobilizer = 0; // input.GetPropertyModifier(AggregateField.effect_massivness_speed_max_modifier).Value;
+
+
+            // Targeting
+            SensorStrength = input.SensorStrength; // input.GetPropertyModifier(AggregateField.sensor_strength).Value;
+
+            //  Same as input.GetPropertyModifier(AggregateField.locking_range).Value * 10;
+            LockingRange = input.MaxTargetingRange * 10; 
+            LockingTime = input.GetPropertyModifier(AggregateField.locking_time).Value / 1000;
```

Comment:
The `AggregateField` should have things like display units or display multiplier, which are transforms on the number for the purposes of rendering in a view for users.